documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -991,3 +991,121 @@ Further ports can be controlled by the DT, for more details see <<part3,section
 ==== Further help
 
 If you've read through this document and not found the answer to a Device Tree problem, there is help available. The author can usually be found on Raspberry Pi forums, particularly the https://forums.raspberrypi.com/viewforum.php?f=107[Device Tree] forum.
+
+[[part5.0]]
+=== Firmware parameters
```

Comment:
I'm not sure if it makes sense for this new "Firmware parameters" section to come _after_ the "Troubleshooting" section?

Reply:
swapped section 4 and 5